source/includes/_notifications.md

```diff
@@ -234,6 +237,7 @@ name       | string  | Name for notification
 entity_type| string  | Entity type for notification
 action     | string | Notification action [account_created, recover_password, attendee_created, attendee_rejected, attendee_confirmed, welcome_email]
 conditions  | json | Conditions for notifications
+mass_notification  | booleand | Flag to identify mass notification
```

Comment:
boleand o **boolean**?, 

la descripción puede ser: 
Flag to identify if the notification that is going to be created is massive

Reply:
Done